easybuild/easyconfigs/m/MetaBAT/MetaBAT-2.14-gompi-2019a.eb

```diff
@@ -0,0 +1,44 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CMakeMake'
+
+name = 'MetaBAT'
+version = '2.14'
+
+homepage = 'https://bitbucket.org/berkeleylab/metabat'
+description = "An efficient tool for accurately reconstructing single genomes from complex microbial communities"
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+
+source_urls = ['https://bitbucket.org/berkeleylab/metabat/get/']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    'd43d5e91afa8f2d211a913739127884669516bfbed870760597fcee2b513abe2',  # v2.14.tar.gz
+]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('CMake', '3.13.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Boost', '1.70.0'),
+    ('zlib', '1.2.11'),
+    ('SAMtools', '1.9'),
+    ('XZ', '5.2.4'),
+    ('bzip2', '1.0.6'),
+]
+
+prebuildopts = "cp -a $EBROOTSAMTOOLS samtools && "
+
+sanity_check_paths = {
+    'files': ['bin/aggregateBinDepths.pl', 'bin/aggregateContigOverlapsByBin.pl', 'bin/metabat', 'bin/metabat2'],
```

Comment:
We should have a Perl dependency here i think

Reply:
Good point, `Perl` should indeed be added as a dependency.

And we should make sure the right `perl` command is used by these scripts as well, by also using:

```python
fix_perl_shebang_for = ['bin/*.pl']